src/main/java/org/geysermc/discordbot/tags/TagsManager.java

```diff
@@ -106,7 +106,7 @@ private static void loadTags() {
                                 switch (lineParts[0]) { // intentional fallthrough
                                     case "type", "aliases" -> tagData.put(lineParts[0], lineParts[1].trim().toLowerCase());
                                     case "image" -> tagData.put(lineParts[0], lineParts[1].trim());
-                                    case "issues" -> issueTriggers = lineParts[1].split("\\|\\|");
+                                    case "issues" -> issueTriggers = lineParts[1].split("\\|\\|"); // Issues are delimited by ||
```

Comment:
Not sure this is needed. Also why did we use a `||` as a delimiter?

Reply:
When I created it, it just seemed like something that would have a really small chance of being in an error we would want to list. Open to suggestions